Source/setmaps.cpp

```diff
@@ -190,6 +191,33 @@ void LoadSetMap()
 		DRLG_SetMapTrans("Levels\\L1Data\\Vile1.DUN");
 		InitNoTriggers();
 		break;
+	case SL_TEST:
+		char dunbuf[100];
+		char palbuf[100];
+		sprintf(palbuf, "derp%d.pal", paletteNum);
+		sprintf(dunbuf, "derp%d.dun", tilesetNum);
+		if (tilesetNum == 0) {
+			LoadPreL1Dungeon(dunbuf, 0, 0);
+			LoadL1Dungeon(dunbuf, 30, 30);
+			AddL1Objs(0, 0, MAXDUNX, MAXDUNY);
+		}
+		if (tilesetNum == 1) {
+			LoadPreL2Dungeon(dunbuf, 0, 0);
+			LoadL2Dungeon(dunbuf, 30, 30);
+			AddL2Objs(0, 0, MAXDUNX, MAXDUNY);
+		}
+		if (tilesetNum == 2) {
+			LoadPreL3Dungeon(dunbuf, 0, 0);
+			LoadL3Dungeon(dunbuf, 30, 30);
+			AddL3Objs(0, 0, MAXDUNX, MAXDUNY);
+		}
+		if (tilesetNum == 3) {
+			LoadPreL4Dungeon(dunbuf, 0, 0);
+			LoadL4Dungeon(dunbuf, 53, 78);
+		}
+		LoadPalette(palbuf);

```

Comment:
You need to call DRLG_SetMapTrans(dunbuf); here for transparancy to work correctly.